framework/rbac/DbManager.php

```diff
@@ -118,8 +126,8 @@ public function init()
      */
     public function checkAccess($userId, $permissionName, $params = [])
     {
-        $assignments = $this->getAssignments($userId);
         $this->loadFromCache();
+        $assignments = $this->getAssignments($userId);
```

Comment:
Currently this PR does not use the cache component for assignments but I has before, so this is a leftover. It should not have any side effects.